k8s/bases/cse-export-d2cc/cse-export-d2cc-task-manager-deployment.yaml

```diff
@@ -79,7 +79,8 @@ spec:
               subPath: application.yml
               name: cse-export-d2cc-task-manager-configmap-volume
           livenessProbe:
-            tcpSocket:
+            httpGet:
+              path: /actuator/health
```

Comment:
since spring 2.3 the healthcheck has two different endpoints for health and for liveness. source: https://spring-gcp.saturnism.me/deployment/kubernetes/health-checks

This said, this isn't enough, we have to find a way to tightly couple the rabbit/postgresql failures to task manager (spring boot app stays up even if they are down, it is not something that comes by default with spring, so we have to implement this behaviour our selves )

Reply:
Out of the box spring boot  health endpoint checks rabbit-mq and database connectivity, if they are down the endpoint answers 503, kubernetes would then restart the application referencing this liveness probe.